db/seeds.rb

```diff
@@ -142,6 +142,34 @@ def create_google_accounts
   end
 end
 
+def create_studio_assesments 
+  1.upto(NUM_STUDIO_ASSESSMENTS) do |i|
+    StudioAssessment.create!(
+      bigpicture_score: Faker::Number.between(from: 0, to: 3),
+      bigpicture_comment: Faker::Cannabis.buzzword,
+      progfundamentals_score: Faker::Number.between(from: 0, to: 3),
+      progfundamentals_comment: Faker::Cannabis.buzzword,
+      versioncontrol_score: Faker::Number.between(from: 0, to: 3),
+      versioncontrol_comment: Faker::Cannabis.buzzword,
+      react_score: Faker::Number.between(from: 0, to: 3),
+      react_comment: Faker::Cannabis.buzzword,
+      node_score: Faker::Number.between(from: 0, to: 3),
+      node_comment: Faker::Cannabis.buzzword,
+      db_score: Faker::Number.between(from: 0, to: 3),
+      db_comment: Faker::Cannabis.buzzword,
+      problemsolving_score: Faker::Number.between(from: 0, to: 3),
+      problemsolving_comment:Faker::Cannabis.buzzword,
+      problemsolvingalt_score: Faker::Number.between(from: 0, to: 3),
+      problemsolvingalt_comment:Faker::Cannabis.buzzword,
+      capstone_passed: Faker::Boolean.boolean,
```

Comment:
Once you merge master into your branch and run `rails db:migrate` you're going to have to modify this line as well, we changed it to "passed_capstone" instead of "capstone_passed"